examples/nodejs/token-vending-machine/lambda/authorizer/package.json

```diff
@@ -4,11 +4,8 @@
   "description": "Momento token vending machine API Gateway Lambda authorizer for the Lambda function",
   "main": "index.ts",
   "scripts": {
-    "prebuild": "eslint . --ext .ts",
     "build": "tsc",
-    "test": "jest",
-    "lint": "eslint . --ext .ts",
-    "format": "eslint . --ext .ts --fix"
+    "test": "jest"
   },
```

Comment:
these were never being exercised and are failing. for the time being, i didnt invest in wiring up eslint everywhere but have tsc in almost all places working